Rubberduck.Core/UI/Command/Refactorings/Notifiers/ExtractInterfaceFailedNotifier.cs

```diff
@@ -22,6 +22,9 @@ protected override string Message(RefactoringException exception)
                         invalidDeclarationType.TargetDeclaration.QualifiedModuleName,
                         invalidDeclarationType.TargetDeclaration.DeclarationType,
                         DeclarationType.ClassModule);
+                case SuspendParserFailureException suspendParserFailure:
+                    Logger.Warn(suspendParserFailure);
+                    return Resources.RubberduckUI.RefactoringFailure_SuspendParserFailure;
```

Comment:
This is outside the PR's scope and would be another PR -- I want to note that I think that this should be in the base. However, the problem seems to be that the `Messge` gets overridden, rather than extended, so we are duplicating the common handling. In future, we should enhance so that `Message` can be extended, allowing the notifiers to focus on specialized concerns.

Reply:
I've moved the handling of this exception case to `RefactoringFailureNotifierBase`.